platform/lib/build/samplesBuilder.js

```diff
@@ -140,33 +143,45 @@ class SamplesBuilder {
       stream = stream.pipe(through.obj(async (sample, encoding, callback) => {
         try {
           this._log.await(`Building sample ${sample.relative} ...`);
-          const parsedSample = await this._parseSample(sample);
-
-          // Skip samples that are drafts for alle envs except development
-          if (parsedSample.document.metadata.draft && config.environment !== 'development') {
-            callback();
-            return;
-          }
-
-          if (!parsedSample.document.metadata.disablePlayground &&
-              !parsedSample.document.metadata.drafts) {
-            this._addToSitemap(sample, parsedSample);
-          }
-
-          // Build various documents and sources that are needed for Grow
-          // to successfully render the example and for the playground
-          const files = [
-            ...this._createDocumentation(sample, parsedSample),
-            ...this._buildRawSources(sample, parsedSample),
-            ...this._createPreview(sample, parsedSample),
-            ...this._renderEmbed(sample, parsedSample),
-          ];
-
-          // Since stream.push doesn't allow to push multiple files at once
-          /* eslint-disable guard-for-in */
-          for (const file of files) {
-            stream.push(file);
+          const samples = [];
+          if (sample.relative.indexOf('1.components/') === -1) {
```

Comment:
why not all samples?

Reply:
It pulls in some unexpected things, e.g. https://amp.dev/documentation/examples/guides/using_the_google_amp_cache/ is somehow an example.

In theory, this is temporary, but fixing it needs to either (1) make sure only examples are examples or (2) make an opt-out mechanism.

I'll add a comment to explain it.

Reply:
You can check if any of these is true:

```
hideCode: true
disablePlayground: true
hidePreview: true
```

That should filter out the weird ones...